packages/nitro-protocol/contracts/interfaces/IMultiAssetHolder.sol

```diff
@@ -40,6 +40,37 @@ interface IMultiAssetHolder {
         uint256[] memory indices
     ) external;
 
+    /**
+     * @param assetIndex Will be used to slice the outcome into a single asset outcome.
```

Comment:
Good catch. Unfortunately these comments are a bit of an abuse of the natspec system -- it would be nice if the compiler picked up mismatches as it would with function parameter docstrings.